src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -93,18 +92,12 @@ export const makeStudyTableHeaders = (datasets: DatasetTerm[], selected: number[
         const isSelectableStudy = datasets.filter(isSelectable).length === datasets.length;
         const tooltipText = 'This study contains one or more dataset that is either open access or whose requests are managed outside DUOS. Go to the datasets tab for additional information per dataset.';
         return {
-          data: <>
-            <ReactTooltip
-              place={'top'}
-              disable={isSelectableStudy}
-              effect={'solid'}
-              scrollHide={true}
-              id={checkboxId}><div style={tooltipStyle}>{tooltipText}</div></ReactTooltip>
-            <div data-for={checkboxId} data-tip={true}>
-              <Checkbox checked={fullySelected} indeterminate={indeterminate} disabled={!isSelectableStudy}
-                onClick={() => onSelect(fullySelected ? _.without(selected, ...studyDatasetIds) : indeterminate ? _.xor(_.without(selected, ...studyDatasetIds), studyDatasetIds) : [...selected, ...studyDatasetIds])}/>
-            </div>
-          </>,
+          data: <div data-for={checkboxId}>
+              <span title={isSelectableStudy ? '' : tooltipText}>
+                  <Checkbox checked={fullySelected} indeterminate={indeterminate} disabled={!isSelectableStudy}
+                            onClick={() => onSelect(fullySelected ? _.without(selected, ...studyDatasetIds) : indeterminate ? _.xor(_.without(selected, ...studyDatasetIds), studyDatasetIds) : [...selected, ...studyDatasetIds])}/>
+              </span>
+          </div>,
```

Comment:
Just want to make sure I understand: Is this removing the tooltip in this case?

Reply:
No, this PR doesn't remove any tooltips; it replaced the rect tooltips, which are rendered as react components, with ones that the browser itself shows. In Safari, this tooltip was already shown in addition to the react tooltip. Using a `span` makes Chrome also show the tooltip, so its behavior now matches Safari's.

To make this case work, I used an empty tooltip text instead of using `disable`:

before:
```
            <ReactTooltip
              place={'top'}
              disable={isSelectableStudy}
```

after:
```
              <span title={isSelectableStudy ? '' : tooltipText}>
```

The logic here is that the tooltip is only shown in the study is not selectable. I've tested this out and it works when running locally.